src/openrct2/interface/Widget.h

```diff
@@ -60,6 +61,62 @@ enum
 
 constexpr uint8_t SCROLLBAR_WIDTH = 10;
 
+constexpr rct_widget MakeWidget(
+    ScreenCoordsXY origin, ScreenSize size, uint8_t type, uint8_t colour, uint32_t content = 0xFFFFFFFF,
+    rct_string_id tooltip = STR_NONE)
+{
+    rct_widget out = {};
+    out.left = origin.x;
+    out.right = origin.x + size.width - 1;
+    out.top = origin.y;
+    out.bottom = origin.y + size.height - 1;
+    out.type = type;
+    out.colour = colour;
+    out.content = content;
+    out.tooltip = tooltip;
+
+    return out;
+}
+
+constexpr rct_widget MakeRemapWidget(
+    ScreenCoordsXY origin, ScreenSize size, uint8_t type, uint8_t colour, uint32_t content = 0xFFFFFFFF,
```

Comment:
I wonder if we shouldn't group `ScreenCoordsXY` and `ScreenSize` into some struct. Maybe `WindowStub` or something alike?

Reply:
Hmm, I rather like having them both separately, to be honest. Easy to pass them on separately to rendering functions that way. I'm also worried that it would impede readability. For example, if WindowStub is only going to contain a `ScreenCoordsXY` and `ScreenSize`, that'd make the initialisation look like `{ {54, 32}, {16, 16} }` or something.

Reply:
Let's keep them separate.